src/NBitcoin/ChainedBlock.cs

```diff
@@ -356,7 +356,7 @@ public DateTimeOffset GetMedianTimePast()
         }
 
         /// <summary>
-        /// Check PoW and that the blocks connect correctly.
+        /// Check PoW/PoS and that the blocks connect correctly.
         /// </summary>
         /// <param name="network">The network being used.</param>
         /// <returns>True if PoW is correct.</returns>
```

Comment:
ee

Reply:
Typo or is another Apro-ism like chmmm?

Reply:
ee is the sound my baby does when she does not like it :)

what i was commenting about was that return value - in summary you have PoW/PoS but in returns tag you have PoW only (so the idea was to have returns to `<c>true</c> if PoW/PoS is correct.` to have it consistent with summary, but see below) 

anyway, this whole comment is little under-descriptive in the sense that what does it really mean to check PoW/PoS and the method does something else really

so i'd go with summary
```
Check that the header is a valid block header including the work done for PoW blocks.
```

and returns should be 
```
<c>true</c> if the header is a valid block header, <c>false</c> otherwise.
```

BTW, However, I've now looked at where exactly this method is being called and I found this:

Look at CHB, when we receive new headers, we call
```
                    bool validated = this.Chain.GetBlock(tip.HashBlock) != null || tip.Validate(this.AttachedNode.Network);
```

so either the tip (the new header that arrived) is on the best chain OR we call this Validate method

now we do 

```
            if (Block.BlockSignature)
                return BlockStake.Validate(network, this);
```

so i assume this is set when we are on stratis with POS blocks, so we go there to validate POS header

```
		public static bool Validate(Network network, ChainedBlock chainedBlock, StakeChain stakeChain = null)
```

and here it seems that stakeChain is always null, so the method can be simplified. But that is offtopic for this PR, so no worries about it.